src/server/pfs/drive/driver.go

```diff
@@ -947,7 +947,7 @@ func (d *driver) inspectFile(file *pfs.File, filterShard *pfs.Shard, shard uint6
 	for commit != nil && (from == nil || commit.ID != from.ID) {
 		diffInfo, ok := d.diffs.get(client.NewDiff(commit.Repo.Name, commit.ID, shard))
 		if !ok {
-			return nil, nil, fmt.Errorf("diff %s/%s/%d not found", commit.Repo.Name, commit.ID, shard)
+			return nil, nil, pfsserver.ErrCommitNotFound
```

Comment:
Seems like we are losing information here by making this change.  Can we make ErrCommitNotFound a struct that contains repo name, commit ID, and shard, and implement the error interface so it can be used as an error?